### DISCUSSION TITLE
Longformer and BERTopic

### DISCUSSION THREAD

**Discussion by Marakyh** [MaartenGr/BERTopic#1473] [Q&A]
Hi! I was wondering why large documents should be divided into sentences/paragraphs instead of using a longformer such as https://huggingface.co/allenai/longformer-base-4096 as a transformer to create the embeddings? 
Thanks!

**Comment by MaartenGr** [comment]
That is because the entire document will be clustered, and as such assigned only a single topic. Instead, if you were to create sentences/paragraphs, multiple topics will be generated from a single document. You can find more about the underlying procedure of BERTopic [here](https://maartengr.github.io/BERTopic/algorithm/algorithm.html).

**Reply by Marakyh** [reply]
Thank you!
